Conduit/Conduit.Db/Follower.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Conduit.Db
+{
+    public class Follower
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.None)]
```

Comment:
no need for this attribute, actually when you create new object just assign new Guid() to the DB model